### DISCUSSION TITLE
Cannot install azuracast on Ubuntu terminal server

### DISCUSSION THREAD

**Discussion by iDavyNL** [AzuraCast/AzuraCast#6203] [Q&A]
Hello, 

We are having issues installing Azuracast. It gives us the following error:
user@ubuntu:/var/azuracast$ sudo su curl -fsSL https://raw.githubusercontent.com/AzuraCast/AzuraCast/main/docker.sh > docker.sh
-bash: docker.sh: Permission denied

It keeps saying permission denied what ever I try, Can someone help me out?

**Comment by scysys** [comment]
You cant use "sudo su" with one line command in that way. Type first

`sudo su`

then do it like here: https://docs.azuracast.com/en/getting-started/installation/docker
